pkg/limayaml/default.yaml

```diff
@@ -25,34 +29,34 @@ images:
 
 # CPUs: if you see performance issues, try limiting cpus to 1.
 # Default: 4
-cpus: 4
+cpus: null
 
 # Memory size
 # Default: "4GiB"
-memory: "4GiB"
+memory: null
 
 # Disk size
 # Default: "100GiB"
-disk: "100GiB"
+disk: null
 
 # Expose host directories to the guest, the mount point might be accessible from all UIDs in the guest
 # Default: none
 mounts:
   - location: "~"
     # CAUTION: `writable` SHOULD be false for the home directory.
     # Setting `writable` to true is possible, but untested and dangerous.
-    writable: false
+    writable: null
     # SSHFS has an optional flag called 'follow_symlinks'. This allows mounts
     # to be properly resolved in the guest os and allow for access to the
     # contents of the symlnk. This defaults to false if not supplied. As a result,
```

Comment:
Can we add “Default: false” line?
The default value is already explained in the existing comment lines, but the format is not consistent with other properties, so hard to find.